<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaneLog</class>
 <widget class="QWidget" name="PlaneLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plane Log</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>911</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidget">
      <property name="font">
       <font>
        <family>Courier</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>AES</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>REG</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>First heard</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Last heard</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Last Few Messages</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../jaero.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear contents</string>
   </property>
  </action>
  <action name="actionUpDown">
   <property name="icon">
    <iconset resource="../jaero.qrc">
     <normaloff>:/images/arrow-up-down-512px.svg</normaloff>:/images/arrow-up-down-512px.svg</iconset>
   </property>
   <property name="text">
    <string>UpDown</string>
   </property>
   <property name="toolTip">
    <string>Expand/Contract vertically</string>
   </property>
  </action>
  <action name="actionLeftRight">
   <property name="icon">
    <iconset resource="../jaero.qrc">
     <normaloff>:/images/arrow-right-left-512px.svg</normaloff>:/images/arrow-right-left-512px.svg</iconset>
   </property>
   <property name="text">
    <string>LeftRight</string>
   </property>
   <property name="toolTip">
    <string>Expand/Contract horizontally</string>
   </property>
  </action>
  <action name="actionStopSorting">
   <property name="icon">
    <iconset resource="../jaero.qrc">
     <normaloff>:/images/stopsort-512px.svg</normaloff>:/images/stopsort-512px.svg</iconset>
   </property>
   <property name="text">
    <string>StopSorting</string>
   </property>
   <property name="toolTip">
    <string>Stop sorting</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../jaero.qrc"/>
 </resources>
 <connections/>
</ui>
